docs/operations/operations.md

```diff
@@ -137,6 +138,7 @@ The default behavior is that, if found, the regional (and/or domain restricted)
 If no entry for the given region exists then the fallback value is the most matching entry (w.r.t. wildcard matching) in the list without a `region` field (or the `keystoneURL` value for the keystone URLs).
 If an additional floating pool should be selectable for a region and/or domain, you can mark it as non constraining
 with setting the optional field `nonConstraining` to `true`.
+Multiple `loadBalancerProviders` can be specified with or without a `region`, also multiple for the same `region`. If a region specific entry is found none of the default entries apply, but the same `name` can be specified with and without a `region`.
```

Comment:
```suggestion
Multiple `loadBalancerProviders` can be specified in the `CloudProfile`. Each provider may specify a region constraint for where it can be used.
If at least once region specific entry exists in the `CloudProfile`, the shoot's specified `loadBalancerProvider` must adhere to the list of the available providers of that region. Otherwise, one of the non-regional specific providers should be used.
Each entry in the `loadBalancerProviders` must be uniquely identified by its name and if applicable, its region.
```